From c5db7af4dee09016489185e26af66cf2f4725a21 Mon Sep 17 00:00:00 2001
From: Dongdong Liu <liudongdong3@huawei.com>
Date: Tue, 9 Mar 2021 21:35:19 +0800
Subject: [PATCH openEuler-22.03-LTS 2/3] lspci: Update tests files with VF
 10-Bit Tag Requester

Update the tests files with the new field 10BitTagReq
in SR-IOV Capabilities Register.

Signed-off-by: Dongdong Liu <liudongdong3@huawei.com>
Signed-off-by: Wangming Shao <shaowangming@h-partners.com>
---
 tests/cap-dvsec-cxl | 4 ++--
 tests/cap-ea-1      | 4 ++--
 tests/cap-pcie-2    | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/tests/cap-dvsec-cxl b/tests/cap-dvsec-cxl
index c499d0e..a24e3fb 100644
--- a/tests/cap-dvsec-cxl
+++ b/tests/cap-dvsec-cxl
@@ -79,8 +79,8 @@
                 PTMControl: Enabled:- RootSelected:-
                 PTMEffectiveGranularity: Unknown
         Capabilities: [b80 v1] Single Root I/O Virtualization (SR-IOV)
-                IOVCap: Migration-, Interrupt Message Number: 000
-                IOVCtl: Enable- Migration- Interrupt- MSE- ARIHierarchy-
+                IOVCap: Migration- 10BitTagReq- Interrupt Message Number: 000
+                IOVCtl: Enable- Migration- Interrupt- MSE- ARIHierarchy- 10BitTagReq-
                 IOVSta: Migration-
                 Initial VFs: 6, Total VFs: 6, Number of VFs: 0, Function Dependency Link: 00
                 VF offset: 16, stride: 2, Device ID: 0d52
diff --git a/tests/cap-ea-1 b/tests/cap-ea-1
index df88ba9..776839d 100644
--- a/tests/cap-ea-1
+++ b/tests/cap-ea-1
@@ -57,8 +57,8 @@
 		ARICtl:	MFVC- ACS-, Function Group: 0
 	Capabilities: [108 v1] Vendor Specific Information: ID=00a0 Rev=1 Len=040 <?>
 	Capabilities: [180 v1] Single Root I/O Virtualization (SR-IOV)
-		IOVCap:	Migration-, Interrupt Message Number: 000
-		IOVCtl:	Enable+ Migration- Interrupt- MSE+ ARIHierarchy+
+		IOVCap:	Migration- 10BitTagReq- Interrupt Message Number: 000
+		IOVCtl:	Enable+ Migration- Interrupt- MSE+ ARIHierarchy+ 10BitTagReq-
 		IOVSta:	Migration-
 		Initial VFs: 128, Total VFs: 128, Number of VFs: 128, Function Dependency Link: 00
 		VF offset: 1, stride: 1, Device ID: a034
diff --git a/tests/cap-pcie-2 b/tests/cap-pcie-2
index 47c8953..9bde139 100644
--- a/tests/cap-pcie-2
+++ b/tests/cap-pcie-2
@@ -48,8 +48,8 @@
 		ARICap:	MFVC- ACS-, Next Function: 1
 		ARICtl:	MFVC- ACS-, Function Group: 0
 	Capabilities: [160] Single Root I/O Virtualization (SR-IOV)
-		IOVCap:	Migration-, Interrupt Message Number: 000
-		IOVCtl:	Enable+ Migration- Interrupt- MSE+ ARIHierarchy-
+		IOVCap:	Migration- 10BitTagReq- Interrupt Message Number: 000
+		IOVCtl:	Enable+ Migration- Interrupt- MSE+ ARIHierarchy- 10BitTagReq-
 		IOVSta:	Migration-
 		Initial VFs: 8, Total VFs: 8, Number of VFs: 1, Function Dependency Link: 00
 		VF offset: 384, stride: 2, Device ID: 10ca
-- 
2.33.0

